RELEASE.md

```diff
@@ -88,26 +88,43 @@ Please make sure you have these tools installed before starting this process:
 
 -   [ ] 8.  Test
 
-    -   Validate that the features present in changelog generated during the circleci pipeline work according to the requirements.
+Involve both the QA and Solutions Engineers stakeholders in the release process.
 
-    -   Do [exploratory tests](https://handbook.dev.codacy.org/product/engineering/QA/levels.html#exploratory-testing) around the functionalities your feature impacted to make sure everything is running as it should.
+    -   Sync with the release manager to identify critical areas of the product affected by the release.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[code-block-style] Code blocks should be fenced](https://app.codacy.com/gh/codacy/chart/pullRequest?prid=5585165)